classes/locations.php

```diff
@@ -308,6 +311,8 @@ class='updated'
 		function additional_information( $post ) {
 			// Additional Information.
 
+			    wp_nonce_field( 'sm_save_postmeta_nonce_action', 'sm_save_postmeta_nonce_field' );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->